packages/styled-components/src/constructors/test/createGlobalStyle.test.js

```diff
@@ -301,7 +301,7 @@ describe(`createGlobalStyle`, () => {
     expect(getCSS(document).trim()).toMatchInlineSnapshot(`"body{background:red;}"`);
 
     render(<A />);
-    expect(getCSS(document).trim()).toMatchInlineSnapshot(`""`);
+    expect(getCSS(document).trim()).toMatchInlineSnapshot(`"body{background:;}"`);
```

Comment:
This was interesting, it should have been rendering this previously and wasn't...